barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -7,6 +7,111 @@
 
 namespace bb {
 
+/*! \brief The implementation of the sum-check Prover for the statements of the form \f$\sum_{\vec \ell \in \{0,1\}^d}
+pow_{\beta}(\vec \ell) \cdot F \left(P_1(\vec \ell),\ldots, P_N(\vec \ell) \right)  = 0 \f$ for multi-linear polynomials
```

Comment:
```suggestion
pow_{\beta}(\vec \ell) \cdot F \left(P_1(\vec \ell),\ldots, P_N(\vec \ell) \right)  = 0 \f$ given multilinear polynomials
```
I think you don't need the dash in multilinear